docs/storage/disks_and_volumes.md

```diff
@@ -1902,23 +1905,18 @@ If you are using local devices or RWO PVCs, setting the affinity on the VMs that
 `Virtiofs` is a shared file system that lets VMs access a directory tree on the host.
 Further details can be found at [Official Virtiofs Site](https://virtio-fs.gitlab.io/).
 
-### Non-Privileged and Privileged Sharing Modes
-
-KubeVirt supports two PVC sharing modes: non-privileged and privileged.
-
-The **non-privileged mode** is enabled by default. This mode has the advantage of not requiring any
-administrative privileges for creating the VM. However, it has some limitations:
+KubeVirt only supports to share PVCs in non-privileged sharing mode. The **non-privileged mode** sharing mode has some
+limitations:
```

Comment:
Since the "privileged mode" doesn't exist anymore, I suggest avoiding mention "non-privileged sharing mode" because it implies that exist another mode. So, maybe something like:
"KubeVirt only supports to share PVCs using a non-privileged virtiofs daemon. Because the lack of privileges, it has some limitations:"

Reply:
You right, nice catch. Done.